baseline-error-prone/src/test/java/com/palantir/baseline/errorprone/JooqResultStreamLeakTest.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.baseline.errorprone;
+
+import com.google.errorprone.BugCheckerRefactoringTestHelper;
+import com.google.errorprone.CompilationTestHelper;
+import org.junit.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+
+@Execution(ExecutionMode.CONCURRENT)
+public final class JooqResultStreamLeakTest {
+
+    private final CompilationTestHelper testHelper =
+            CompilationTestHelper.newInstance(JooqResultStreamLeak.class, getClass());
+
+    private final BugCheckerRefactoringTestHelper refactoringTestHelper =
+            BugCheckerRefactoringTestHelper.newInstance(new JooqResultStreamLeak(), getClass());
```

Comment:
one more nit: mind using our `RefactoringValidator` instead of `BugCheckerRefactoringTestHelper`? Ours validates that the refactored result also passes validation